lib/srv/db/server.go

```diff
@@ -713,15 +727,25 @@ func (s *Server) startServiceHeartbeat() error {
 			return nil, trace.Wrap(err)
 		}
 
+		instanceMD, err := s.cfg.CloudInstanceMetadataClient.GetInstanceMetadata(ctx)
+		if err != nil {
+			return nil, trace.Wrap(err)
+		}
```

Comment:
If for some reasons the `InstanceMetadataClient.GetInstance` will fail the whole `Heartbeat` will be affected. Wonder if in that case we should should fallback to empty Metadata struct.